pkg/resource/certificate_authority_activation/hooks.go

```diff
@@ -45,7 +45,7 @@ func (rm *resourceManager) customFindCertificateAuthorityActivation(
 	// If any required fields in the input shape are missing, AWS resource is
 	// not created yet. Return NotFound here to indicate to callers that the
 	// resource isn't yet created.
-	if r.ko.Spec.CertificateAuthorityARN == nil {
+	if r.ko.Spec.CertificateAuthorityARN == nil && r.ko.Spec.CertificateAuthorityRef.From.Name == nil {
```

Comment:
One of those fields should be provided to be able to create a CA. https://github.com/aws-controllers-k8s/acmpca-controller/blob/main/pkg/resource/certificate_authority_activation/sdk.go#L111-L113.